### DISCUSSION TITLE
How to use failed_jobs?

### DISCUSSION THREAD

**Discussion by Restingo** [romanzipp/Laravel-Queue-Monitor#75] [Q&A]
How can I use `failed_jobs` as stated in the README? So that this package only keeps and informs me about jobs with errors?

**Comment by romanzipp** [comment]
The `failed_jobs` table is a [built-in Laravel feature](https://laravel.com/docs/8.x/queues#dealing-with-failed-jobs) which only collects information about failed jobs. The Queue-Monitor package mimics this functionality using the [described method](https://github.com/romanzipp/Laravel-Queue-Monitor#only-keep-failed-jobs)
